synapse/lib/agenda.py

```diff
@@ -403,6 +403,9 @@ def unpack(cls, stor, val):
         appt.lastfinishtime = val['lastfinishtime']
         appt.lastresult = val['lastresult']
         appt.enabled = val['enabled']
+        appt.startcount = val['startcount']
+        appt.errcount = val['errcount']
+        appt.lasterrs = val['lasterrs']
```

Comment:
These values may not be present on an existing storage record, so we have to use `.get()` on them with a default value.